posix.mak

```diff
@@ -964,4 +966,25 @@ $G/contributors_list.ddoc:  | $(STABLE_RDMD) $(TOOLS_DIR) $(INSTALLER_DIR)
 $(DDOC_BIN): ddoc.d | $(STABLE_DMD)
 	$(STABLE_DMD) -of$@ $<
 
+################################################################################
+# Build and render the DMD man page
+# ---------------------------------
+#
+# This allows previewing changes to the automatically generated DMD man page
+################################################################################
+
+$W/$(MAN_PAGE): $(DMD_DIR)/docs/gen_man.d $(DMD_DIR)/src/dmd/cli.d | ${STABLE_DMD}
+	${MAKE} -C $(DMD_DIR)/docs DMD=$(abspath $(STABLE_DMD)) build
+	# Remove the loop after https://github.com/dlang/dmd/pull/7637 has been merged
+	for file in "$(DMD_DIR)/generated/$(MAN_PAGE)" "$(DMD_DIR)/$(MAN_PAGE)" ; do \
+		if [ -f "$$file" ] ; then \
+			mkdir -p $(dir $@); \
+			cp $$file $@; \
+			break; \
+		fi \
+	done
+	${MAKE} -s -C $(DMD_DIR)/docs DMD=$(abspath $(STABLE_DMD)) preview > $(dir $@)dmd.text
```

Comment:
Maybe there's a better extension for such plaintext files?

Reply:
I've become quite accustomed to `.txt`